File: implementation/scipamato/scipamato-core-sync/src/test/java/ch/difty/scipamato/core/sync/jobs/language/LanguageItemWriterTest.java
package ch.difty.scipamato.core.sync.jobs.language;

import org.jooq.DSLContext;

import ch.difty.scipamato.core.sync.jobs.AbstractItemWriterTest;

class LanguageItemWriterTest extends AbstractItemWriterTest<PublicLanguage, LanguageItemWriter> {

    @Override
    protected LanguageItemWriter newWriter(DSLContext dslContextMock) {
        return new LanguageItemWriter(dslContextMock);
    }

}

File: implementation/scipamato/core/sync/src/test/kotlin/ch/difty/scipamato/core/sync/jobs/language/LanguageItemWriterTest.kt
package ch.difty.scipamato.core.sync.jobs.language

import ch.difty.scipamato.core.sync.jobs.AbstractItemWriterTest
import org.jooq.DSLContext

internal class LanguageItemWriterTest : AbstractItemWriterTest<PublicLanguage, LanguageItemWriter>() {
    override fun newWriter(dslContextMock: DSLContext?) = LanguageItemWriter(dslContextMock)
}
